pkg/controller/provider/ova-setup.go

```diff
@@ -26,21 +28,21 @@ const (
 	qemuGroup              = 107
 )
 
-func (r Reconciler) CreateOVAServerDeployment(provider *api.Provider, ctx context.Context) {
+func (r Reconciler) CreateOVAServerDeployment(provider *api.Provider, ctx context.Context) (err error) {
 	ownerReference := metav1.OwnerReference{
 		APIVersion: "forklift.konveyor.io/v1beta1",
 		Kind:       "Provider",
 		Name:       provider.Name,
 		UID:        provider.UID,
 	}
-	pvName := fmt.Sprintf("%s-pv-%s-%s", ovaServer, provider.Name, provider.Namespace)
-	err := r.createPvForNfs(provider, ctx, ownerReference, pvName)
+	pvName := fmt.Sprintf("%s-pv-%s-%s-%s", ovaServer, provider.Name, provider.Namespace, generateRandomSuffix())
```

Comment:
why generating the name ourselves rather than letting k8s do that (by setting `GenerateName`)?

Reply:
because we need it later on for the PVC creation, but I can add a query to get it value

Reply:
no need to query, we should get it the PV back with the content from the server

Reply:
yeah, I accidentally returned `generatedName` instead `name` so the random suffix was missing, now working.